lib/widgets/teach_word/stroke_order_animation_controller.dart

```diff
@@ -0,0 +1,632 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
```

Comment:
Sort the imports in alphabetical order.
Reference : https://dart.dev/effective-dart/style

Reply:
please tell me it's not sorted correctly